.github/ISSUE_TEMPLATE/new-workshop.md

```diff
@@ -0,0 +1,20 @@
+---
+name: New workshop
+about: Checklist for workshop organising
+title: YYYY-MM-DD online/in-person workshop
+labels: ''
+assignees: ''
+
+---
+
+- [ ] create new events in the [training catalogue](http://app.manchester.ac.uk/training/default.aspx)
+- [ ] find helpers for the course
+- [ ] create an Outlook calendar event and invite helpers and
+  ResearchITNews@manchester.ac.uk
+- [ ] set up a Zoom event / book a [teaching room](https://resourcebooker.manchester.ac.uk/)
+- [ ] update the custom reminder email with Zoom/room details (this is sent 1 week before the course)
+- [ ] move applicants from the waiting list onto the new event
```

Comment:
Rather than 'move applicants from the waiting list' how about 'notify waiting list about the upcoming event'?